common/hyperscaler/rules/simple_parser.go

```diff
@@ -0,0 +1,93 @@
+package rules
+
+import (
+	"fmt"
+	"strings"
+)
+
+type SimpleParser struct {
+}
+
+func (g *SimpleParser) Parse(ruleEntry string) (*Rule, error) {
+	outputRule := NewRule()
+
+	ruleEntry = strings.ReplaceAll(ruleEntry, " ", "")
```

Comment:
Could be separated in a function with single responsibility - removal of insignificant characters.